clang/include/clang/Basic/arm_neon.td

```diff
@@ -2115,3 +2115,8 @@ let ArchGuard = "defined(__aarch64__)", TargetGuard = "lut" in {
     def VLUTI4_BF_X2_Q   : SInst<"vluti4_laneq_x2", ".2(<U)I", "Qb">;
   }
 }
+
+let ArchGuard = "defined(__aarch64__)", TargetGuard = "faminmax" in {
```

Comment:
Should this target guard not be `"neon,faminmax"`? We are currently testing the use of these intrinsics under `+neon,+faminmax` and `FeatureFAMINMAX` does not include `FeatureNEON` as a dependency.

Reply:
Done